kuri_navigation/env-hooks/kuri_navigation.sh

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env sh
 
 kuri_key_teleop() {
-    rosrun kuri_navigation keyboard_teleop $@
+    rosrun kuri_teleop keyboard_teleop.py $@
```

Comment:
It's [not well documented](http://wiki.ros.org/PyStyleGuide), but convention is that node executables have `<filename>` == `<node name>`. So any executable node script shouldn't have a file extension. You can observe this is any standard ROS package: https://github.com/ros/geometry/tree/melodic-devel/tf/scripts